tools/gentool/gentool.go

```diff
@@ -168,14 +172,14 @@ func argParse() *CmdParams {
 	if *fieldSignable {
 		cmdParse.FieldSignable = *fieldSignable
 	}
-	return &cmdParse
+	return &cmdParse, nil
 }
 
 func main() {
 	// cmdParse
-	config := argParse()
-	if config == nil {
-		log.Fatalln("parse config fail")
+	config, err := argParse()
+	if err != nil {
+		log.Fatalln("parse config fail\n", err)
```

Comment:
Since `config` is initialized as an empty struct, it never be nil and this error handling was slipped through.
The error message is always `connect db server fail: dsn cannot be empty` since `dsn` is always an empty string when a parse error occurs.

IMO this error message is a bit inappropriate as hint for the user what to do.
After the fix, the error message will look like this with following config file.

#### gen.yml
```
tables: users
```

#### error message
```
2022/10/10 14:45:45 parse config fail
 yaml: unmarshal errors:
  line 5: cannot unmarshal !!str `users` into []string
```